mirdata/mridangam_stroke.py

```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
+
+"""
+Mridangam Stroke Dataset Loader
+
+The Mridangam Stroke dataset is a collection of individual strokes of
+the Mridangam in various tonics. The dataset comprises of 10 different
+strokes played on Mridangams with 6 different tonic values. The audio
+examples were recorded from a professional Carnatic percussionist in a
+semi-anechoic studio conditions by Akshay Anantapadmanabhan.
+
+Total audio samples: 6977
+
+Used microphones:
+* SM-58 microphones
+* H4n ZOOM recorder.
+
+Audio specifications
+* Sampling frequency: 44.1 kHz
+* Bit-depth: 16 bit
+* Audio format: .wav
+
+The dataset can be used for training models for each Mridangam stroke. The
+presentation of the dataset took place on the IEEE International Conference
+on Acoustics, Speech and Signal Processing (ICASSP 2013) on May 2013.
+You can read the full publication here: https://repositori.upf.edu/handle/10230/25756
+
+Mridangam Dataset is annotated by storing the informat of each track in their filenames.
+The structure of the filename is:
+<TrackID>__<AuthorName>__<StrokeName>-<Tonic>-<InstanceNum>.wav
+
+The dataset is made available by CompMusic under a Creative Commons
+Attribution 3.0 Unported (CC BY 3.0) License.
+
+For more details, please visit: https://compmusic.upf.edu/mridangam-stroke-dataset
+"""
+
+import os
+import librosa
+
+from mirdata import download_utils
+from mirdata import jams_utils
+from mirdata import track
+from mirdata import utils
+
+
+DATASET_DIR = 'Mridangam-Stroke'
+
+
+REMOTES = {
+    'remote_data': download_utils.RemoteFileMetadata(
+        filename='mridangam_stroke_1.5.zip',
+        url='https://zenodo.org/record/4068196/files/mridangam_stroke_1.5.zip?download=1',
+        checksum='39af55b2476b94c7946bec24331ec01a',  # the md5 checksum
+        destination_dir=None,  # relative path for where to unzip the data, or None
+    ),
+}
+
+DATA = utils.LargeData(
+    'mridangam_stroke_index.json'
```

Comment:
I assume there isn't metadata right?

Reply:
No! There is just a file for each folder to relate the files with their respective Freesound link (because all the files are uploaded to Freesound).